bedrock/firefox/templates/firefox/ios.html

```diff
@@ -122,6 +124,38 @@
   </div>{#-- /.container --#}
 </section>{#-- /#sync --#}
 
+<section class="section section-privacy" id="privacy">
+  <div class="container">
+    <h2 class="heading">{{ _('Keep your private info private') }}</h2>
+
+    <p class="lead">
+      {{ _('We believe your personal information belongs to you.') }}
+      {{ _('Firefox for iOS is filled with features to help keep it that way. ') }}
+    </p>
+
+    <ul>
+      <li id="private-browsing">
+      {% if waffle.switch('tracking-protection') %}
```

Comment:
I don't think it does but wanted to confirm, hence the waffle switch. I suspect I'll be removing this.